packages/vitest-plugin-storybook/src/global-setup.ts

```diff
@@ -0,0 +1,81 @@
+import { spawn, type ChildProcess } from 'node:child_process'
+import { log } from './utils'
+import http from 'node:http'
+import https from 'node:https'
+
+import type { GlobalSetupContext } from 'vitest/node'
+
+let storybookProcess: ChildProcess | null = null
+
+const checkStorybookRunning = (storybookUrl: string) => {
+  return new Promise<boolean>((resolve) => {
+    const url = new URL(`${storybookUrl}/iframe.html`)
+    const protocol = url.protocol === 'https:' ? https : http
+
+    const options = {
+      method: 'HEAD',
+      host: url.hostname,
+      path: url.pathname,
+    }
+
+    const req = protocol.request(options, (res) => {
+      resolve(res.statusCode === 200)
+    })
+
+    req.on('error', () => resolve(false))
+    req.end()
+  })
+}
+
+const startStorybookIfNeeded = async () => {
```

Comment:
```suggestion
const startStorybookIfNotRunning = async () => {
```
That explains the need.